cdc/owner/owner.go

```diff
@@ -92,6 +93,7 @@ func NewOwner(pdClient pd.Client) *Owner {
 		gcManager:     gc.NewManager(pdClient),
 		lastTickTime:  time.Now(),
 		newChangefeed: newChangefeed,
+		logLimiter:    rate.NewLimiter(1, 1),
```

Comment:
Can you add the rate as a constant?

Reply:
Can you comment on what logs `logLimiter` limits?

Reply:
done